other/materials_designer/import_material_from_jarvis_db_entry.ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
<p>I'd just leave the h2 title only</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>